src/components/cylc/GraphSubgraph.vue

```diff
@@ -0,0 +1,49 @@
+<template>
```

Comment:
Missing the copyright notice. Also could do with a comment explaining what this component is for.